ui/imports/shared/status/StatusStickerMarket.qml

```diff
@@ -38,6 +38,26 @@ Item {
     signal updateClicked(string packId)
     signal buyClicked(string packId)
 
+    QtObject {
+        id: d
+
+        function runSendModal(price, packId) {
+
+            let token = ModelUtils.getByKey(root.walletAssetsStore.groupedAccountAssetsModel, "tokensKey", root.store.stickersStore.getStatusTokenKey())
+
+            root.sendModalPopup.interactive = false
+            root.sendModalPopup.preSelectedRecipient = root.store.stickersStore.getStickersMarketAddress()
+            root.sendModalPopup.preSelectedRecipientType = Helpers.RecipientAddressObjectType.Address
+            root.sendModalPopup.preSelectedHoldingID = !!token && !!token.symbol ? token.symbol : ""
+            root.sendModalPopup.preSelectedHoldingType = Constants.TokenType.ERC20
+            root.sendModalPopup.preSelectedSendType = Constants.SendType.StickersBuy
+            root.sendModalPopup.preDefinedAmountToSend = LocaleUtils.numberToLocaleString(parseFloat(price))
+            root.sendModalPopup.preSelectedChainId = root.store.mainModuleInst.appNetworkId
```

Comment:
Please avoid using context properties directly like in this case `mainModuleInst` it affects testability, even if not now, may be a problem in the future. Stores are intended to be only place where we access ctx props. It also makes the store's API more explicit and cleaner.

Reply:
`mainModuleInst` is a prop in the store, so technically it is not a direct access to the context property. But would you like to have a function in the store that returns `appNetworkId` or? The same question, since it's used in many places of that app, is that for the scope of this PR or not?

Reply:
Ideally the `appNetworkId` should be a property of the store, defined like

```
readonly property string appNetworkId: mainModuleInst.appNetworkId
```
with the `mainModuleInst` private within the store.

Fixing it in this PR would be cool because it's kind of new code (`preSelectedChainId` was not set before).